modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -40,6 +40,9 @@ def validate_veteran_identifiers(require_birls: false)
 
         mpi_add_response = target_veteran.mpi.add_person
         raise mpi_add_response.error unless mpi_add_response.ok?
+
+      rescue ::Common::Exceptions::UnprocessableEntity
+        raise ::Common::Exceptions::UnprocessableEntity.new(detail: 'User is missing EDIPI. Please contact _fill_this_in_ to resolve this issue.')
```

Comment:
I was originally just catching this and re-raising. If EDIPI _is_ required, then we want to communicate a better message to consumers advising how to resolve. If the other changes in this PR are acceptable, then this can get removed